### PR TITLE
cri-tools

### PR DESCRIPTION
**Pull request by danieldop**
default variables
README

**Issue #, if available:**

**Description of changes:**

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Testing Done**

*[See this guide for recommended testing for PRs.](../doc/CONTRIBUTING.md#testing-changes) Some tests may not apply. Completing tests and providing additional validation steps are not required, but it is recommended and may reduce review time and time to merge.*

### CODE REVIEW

File: eks-worker-al2-variables.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "ami_users": "",
     "associate_public_ip_address": "",
     "aws_access_key_id": "{{env `AWS_ACCESS_KEY_ID`}}",
-    "aws_region": "us-west-2",
+    "aws_region": "us-east-1",
```
Comment (dims) [inline eks-worker-al2-variables.json:9]: do you really need this change, seems unrelated to cri-tools

File: eks-worker-al2-variables.json [hunk 2]
```diff
@@ -33,5 +33,5 @@
     "ssh_username": "ec2-user",
     "subnet_id": "",
     "temporary_security_group_source_cidrs": "",
-    "volume_type": "gp2"
+    "volume_type": "gp3"
```
Comment (dims) [inline eks-worker-al2-variables.json:36]: do you really need this change, seems unrelated to cri-tools

File: README.md [hunk 3]
```diff
@@ -7,6 +7,14 @@ EKS-optimized AMI.
 
 **Check out the AMI's [user guide](doc/USER_GUIDE.md) for more information.**
 
+## Excuting ##
```
Comment (dims) [inline README.md:10]: check spelling please!

File: README.md [hunk 4]
```diff
@@ -7,6 +7,14 @@ EKS-optimized AMI.
...
+```shell
+EXPORT AWS_PROFILE=prod
```
Comment (dims) [inline README.md:12]: EXPORT in upper case?

i think you can skip changes to this file for this PR (and open one later for adding the section in another PR)

**Comment by cartermckinnon** [comment]
@danieldop Did you mean to push this to a fork? Please clean up this branch to include only the installation of `cri-tools`.

**Comment by cartermckinnon** [comment]
Closing in favor of #1235 .
